### PR TITLE
[zh-cn]: update the translation of Document `documentElement` property 

### PR DESCRIPTION
**Pull request by T34-active**
and fix svg `<feImage>` element Specifications translation

### CODE REVIEW

File: files/zh-cn/web/api/document/documentelement/index.md [hunk 1]
```diff
@@ -1,38 +1,34 @@
...
-var element = document.documentElement;
-```
+{{domxref("Element")}} 对象。
 
 ## 示例
 
 ```js
 const rootElement = document.documentElement;
 const firstTier = rootElement.childNodes;
+// firstTier 是一个包含根元素直接子元素的 NodeList
```
Comment (T34-active) [inline files/zh-cn/web/api/document/documentelement/index.md:21]:
```suggestion
// firstTier 是包含根元素直接子元素的 NodeList
```

Comment (T34-active) [inline files/zh-cn/web/api/document/documentelement/index.md:21]:
```suggestion
// firstTier 是包含根元素直接子元素的 NodeList
```

**Review by T34-active — COMMENTED** [review]
fix content

File: files/zh-cn/web/api/document/documentelement/index.md [hunk 2]
```diff
@@ -1,38 +1,34 @@
...
+title: Document：documentElement 属性
 slug: Web/API/Document/documentElement
+l10n:
+  sourceCommit: 41a8b9c9832359d445d136b6d7a8a28737badc6b
 ---
 
 {{ApiRef("DOM")}}
 
-**`Document.documentElement`** 是一个会返回文档对象（{{domxref("document")}}）的根{{domxref("Element", "元素")}}的只读属性（如 HTML 文档的 {{HTMLElement("html")}} 元素）。
+**`Document.documentElement`** 返回 {{domxref("document")}} 的根元素 {{domxref("Element")}}（例如，HTML 文档的 {{HTMLElement("html")}} 元素）。
```
Comment (yin1999) [inline files/zh-cn/web/api/document/documentelement/index.md:10]:
```suggestion
**`Document.documentElement`** 返回 {{domxref("document")}} 的根{{domxref("Element", "元素", "", "nocode")}}（例如，HTML 文档的 {{HTMLElement("html")}} 元素）。
```

File: files/zh-cn/web/api/document/documentelement/index.md [hunk 3]
```diff
@@ -1,38 +1,34 @@
...
 ```
 
 ## 备注
 
-对于任何非空 HTML 文档，调用 `document.documentElement` 总是会返回一个 {{HTMLElement("html")}} 元素，且它一定是该文档的根元素。借助这个只读属性，能方便地获取到任意文档的根元素。
-
-HTML 文档通常包含一个子节点 {{HTMLElement("html")}}，但在它前面可能还有个 DOCTYPE 声明。XML 文档通常包含多个子节点：根元素，DOCTYPE 声明，和 [processing instructions](/zh-CN/docs/DOM/ProcessingInstruction)。
-
-所以，应当使用 `document.documentElement` 来获取根元素，而不是 {{Domxref("document.firstChild")}}。
+对于任何非空 HTML 文档，`documentElement` 将始终是 {{HTMLElement("html")}} 元素。对于任何非空 XML 文档，`documentElement` 将始终是文档根元素的任何元素。
```
Comment (yin1999) [inline files/zh-cn/web/api/document/documentelement/index.md:31]:
```suggestion
对于任何非空 HTML 文档，`documentElement` 将始终是 {{HTMLElement("html")}} 元素。对于任何非空 XML 文档，`documentElement` 将始终是文档的根元素。
```

**Review by yin1999 — APPROVED** [review]
感谢 @T34-active

**Pull request merged**